packages/teams_memory/teams_memory/utils/teams_bot_middlware.py

```diff
@@ -41,7 +41,9 @@ def build_deep_link(context: TurnContext, message_id: str):
     return f"https://teams.microsoft.com/l/message/{deeplink_conversation_id}/{message_id}?context=%7B%22contextType%22%3A%22chat%22%7D"
 
 
-class MemoryMiddleware(Middleware):
+class MemoryMiddleware(Middleware):  # type: ignore
```

Comment:
do we keep type ignore?

Reply:
Without it there's this error:
![image](https://github.com/user-attachments/assets/35de4bde-6549-43d2-bc5f-2022a06f6ce7)

Alternative is creating a `SendActivities` Protocol class and casting `responses` - does that work for you?

Reply:
nah, idc. let's type ignore

Reply:
(weird tho)

Reply:
Actually that doesn't work either, I keep getting the same error. Keeping. Feel free to iterate on it!